scripts/taginfo_template.json

```diff
@@ -65,11 +65,141 @@
       "description": "Populates shields that mark roads carrying the route relation.",
       "doc_url": "https://openmaptiles.org/schema/#route_1"
     },
+    {
+      "key": "unsigned_ref",
+      "object_types": ["relation"],
+      "description": "Suppresses any shields that would mark the roads carrying the route relation, unless the network=* consists of unnumbered routes.",
```

Comment:
`unsigned_ref` is interesting because we “support” it by omission. There’s no mention of the key in either this repository or OpenMapTiles; rather, it’s a tagging convention we lend our support to by the fact we render `ref` unconditionally.

Reply:
Pwrhaps unsigned_ref should state something like that it holds a value that would otherwise be in ref if it were signed

Reply:
Taginfo already incorporates the wiki’s definitions of these tags. The purpose of this file is to supplement that definition with how _we_ use the data.

Reply:
Yes, but our description is wrong.  Unsigned_ref suppresses nothing if ref is also set.

Reply:
OK, I see what you mean. I added a clause explaining that `ref` takes precedence over `unsigned_ref`.